### DISCUSSION TITLE
1.0.9.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#40] [Обновления]
# 1.0.9.1
- Добавлена поддержка Twitch Videos & m.twitch.tv
- Добавлена поддержка Youtube Embed
- Функционал перевода вынесен в отдельную функцию

Известные проблемы: #39 

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.0.9...1.0.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9.1'>1.0.9.1</a>.</em>

**Comment by ilyhalight** [comment]
Поддержка остальных платформ появится в ближайшее время.

**Comment by FiLinX** [comment]
обновил скрипт
 к сожалению на десктоп страницах твича  -  панель не появляется(
пример - https://www.twitch.tv/videos/1609057537
 но, если этот же твич открыть в мобильной версии - то панель и перевод работает!
 пример - https://m.twitch.tv/videos/1609057537
 и да, если в мобильной, перейти на любое видео из колонки справа - то панельки нету сразу, необходимо перезагрузить вкладку..
 есть ещё твич-плеер в отдельном окне - https://player.twitch.tv/?enableExtensions=true&muted=false&parent=twitch.tv&player=popout&quality=auto&t=2h10m48s&video=1609057537&volume=0.5   и панельки тут нету..
 вин10-64 LTSC в браузере Iron на 87 хроме 32 бит
 
 зы:  очепятка в посте:  наверное не Twitch embed videos  -  а Youtube embed videos  )
 и да, спасибо - в эмбэд ютубинах перевод работает теперь!
 
 упс.. заметил ещё одну, не знаю, странность чтоль..
 вобщем если кликнуть на троеточие в панельке, то как бы контур кнопки троеточия выезжает в правый бок)
 скрин - https://i.postimg.cc/hvy5FmQT/screenshot-60.png
 но главное  -  это если в браузере уже присутствуют ещё несколько вкладок с плеерами ютуба,  и на любой из них в развёрнутом меню панельки переводчика нажать на "сбросить настройки"  -  то все ютубные вкладки начинают перезагружаться, и если на каких либо из них видео было поставлено на паузу то они все начинают воспроизводиться!
 такая прикольная какофония получается))
 но да, это уже наверное зависит от системы и браузера

**Reply by ilyhalight** [reply]
> к сожалению на десктоп страницах твича - панель не появляется(

Почему-то на записях стримов панелька иногда пропадает. На видео из вкладки "яркие моменты" с таким не сталкивался. Немного позже постараюсь исправить это.
![изображение](https://user-images.githubusercontent.com/62353659/197422911-100df3f2-9fa2-4e6d-a9e5-59845c839c54.png)

> и да, если в мобильной, перейти на любое видео из колонки справа - то панельки нету сразу, необходимо перезагрузить вкладку..

Об этом знаю. Сейчас особо нету идей, как это можно было бы быстро исправить, поэтому буду это фиксить немного позже, когда доделаю адаптацию для остальных сайтов.

 > есть ещё твич-плеер в отдельном окне
 
 Не знал. Постараюсь добавить поддержку и его 👍 
 
 > панельки переводчика нажать на "сбросить настройки" - то все ютубные вкладки начинают перезагружаться, и если на каких либо из них видео было поставлено на паузу то они все начинают воспроизводиться!
 
 Это особенность работы IndexedDB, ну, или моих кривых ручек :)
 
 >  вобщем если кликнуть на троеточие в панельке, то как бы контур кнопки троеточия выезжает в правый бок)
 
 Постараюсь исправить, если получится воспроизвести это.
 
 Спасибо за отзыв =)

**Reply by ilyhalight** [reply]
> к сожалению на десктоп страницах твича - панель не появляется(
>> Почему-то на записях стримов панелька иногда пропадает. На видео из вкладки "яркие моменты" с таким не сталкивался. Немного позже постараюсь исправить это.

Исправил в 1.0.9.2

> есть ещё твич-плеер в отдельном окне
>>Не знал. Постараюсь добавить поддержку и его 👍

Добавлю поддержку в 1.0.9.3... если опять не забуду...

>вобщем если кликнуть на троеточие в панельке, то как бы контур кнопки троеточия выезжает в правый бок)
>>Постараюсь исправить, если получится воспроизвести это.

Исправил в 1.0.9.2

**Comment by alexalsp** [comment]
В опере после обновления нет панели ((

**Reply by ilyhalight** [reply]
> В опере после обновления нет панели ((

Используй версию расширения 1.0.9, постараюсь исправить в ближайшее время

**Reply by ilyhalight** [reply]
> В опере после обновления нет панели ((

А что за версия оперы? Попробовал на GX, всё работает
![изображение](https://user-images.githubusercontent.com/62353659/197872453-7a767ebd-4e3a-4179-8048-18dba474cb67.png)
